apps/sms/views/inbox/test/unit/inbox_test.js

```diff
@@ -667,6 +668,9 @@ suite('thread_list_ui', function() {
 
       assert.isFalse(firstThreadNode.classList.contains('unread'));
       assert.isFalse(secondThreadNode.classList.contains('unread'));
+
+      sinon.assert.neverCalledWith(MessageManager.markThreadRead, 1, true);
```

Comment:
True - u want to mark as read
False - u want to make as unread
IIRC same thing I told on jsdoc bug  also.